sequence_transform.c

```diff
@@ -435,9 +435,9 @@ bool process_sequence_transform(uint16_t keycode, keyrecord_t *record, uint16_t
 
     st_debug(ST_DBG_GENERAL, "pst keycode: 0x%04X, mods: 0x%02X, pressed: %d\n",
         keycode, mods, record->event.pressed);
-    // If this is one of the special keycodes, convert to our internal trie code
-    if (keycode >= special_key_start && keycode < special_key_start + SEQUENCE_TRANSFORM_COUNT) {
-        keycode = keycode - special_key_start + SPECIAL_KEY_TRIECODE_0;
+    // If this is one of the sequence token keycodes, convert to our internal trie code
+    if (keycode >= sequence_token_start && keycode < sequence_token_start + TRIECODE_SEQUENCE_TOKEN_0) {
```

Comment:
BUG: should be `sequence_token_start + SEQUENCE_TRANSFORM_COUNT` !

Reply:
Yes! It slips through without breaking anything, because the expanded keycode range from the bug is unused... but this is a bad one. Sorry!